modules/data.land/R/dataone_download.R

```diff
@@ -24,23 +25,16 @@ dataone_download = function(id, username, CNode = "PROD", lazyLoad = FALSE, quie
   pkg <- dataone::getDataPackage(d1c, id = id, lazyLoad = lazyLoad, quiet = quiet, limit = "1MB") # what is the standard limit for pecan downloads?
   files <- datapack::getValue(pkg, name="sysmeta@formatId")
   n <- length(files) # number of files
-  
-  # fileath to /dbfiles 
-  fp <- "/fs/data1/pecan.data/dbfiles/"
-  
+
   # make new directory within this directory
-  newdir <- paste(fp, "NewData_", username, sep = "")
-  system(paste("mkdir", newdir))
-  
-  # switch to new directory -- unsure if I should do this in R or in unix
-  # system(paste("cd", fp, sep = " ")) 
-  setwd(newdir)
+  newdir <- paste(filepath, "NewData_", username, sep = "")
```

Comment:
Yeah, we want to avoid slashes in folder names. 
One option is to use a gsub to replace all / with _, -, or .
The other would be to basename the id to just the last bit.